test/plugins/features/movies/index.test.js

```diff
@@ -18,6 +28,17 @@ describe('movies integration', () => {
       });
     });
 
+    it('lists movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        query: {}
```

Comment:
Do you need this? My understanding was that Hapi would parse out the query parameters on the `url`.

ie. `/movies?a=1` would parse to `{a: 1}` in `request.query`.